tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.robot

```diff
@@ -98,16 +92,27 @@ Get VCH List Within Invalid Datacenter
     Verify Return Code
     Verify Status Not Found
 
-
 Get VCH List Within Invalid Compute Resource
     Get Path Under Target    vch    compute-resource=INVALID
 
     Verify Return Code
     Verify Status Bad Request
 
-
 Get VCH List Within Invalid Datacenter and Compute Resource
     Get Path Under Target    datacenter/INVALID/vch    compute-resource=INVALID
 
     Verify Return Code
     Verify Status Not Found
+
+   
+Get Empty VCH List When No VCH deployed
+    Run Secret VIC Machine Delete Command   %{VCH-NAME}
+
+    Get VCH List
+
+    Verify Return Code
+    Verify Status Ok
+
+    ${vchs}=  Run  echo '${OUTPUT}' | jq -r '.vchs[]'
+    Log  ${vchs}
+    Length Should Be  ${vchs}  0
```

Comment:
Perhaps these three lines could be captured in a `Verify VCH List Empty` keyword (or similar), for consistency with the rest of the tests.

Reply:
fixed